docs/UserGuide.md

```diff
@@ -117,23 +117,45 @@ Examples:
 * `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 * `edit 3 sch/NJC stream/` Edits the school of the 3rd person to be `NJC` and clears academic stream data.
 
-### Locating persons by name: `find`
+### Finding persons by fields: `find`
 
-Finds persons whose names contain any of the given keywords.
+Finds all persons whose fields match the given keywords.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `find [n/KEYWORD …] [a/KEYWORD …] [e/KEYWORD …] [p/KEYWORD …] `
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+* There should be at least one field i.e. `find` is an invalid command.
```

Comment:
Saying "`find` by itself is an invalid command" might be clearer.
Also, I went to do a bit of research, seems that "... at least one field, i.e., `find`..." with the commas before and after i.e. is more grammatically correct. similarly for e.g.